Libraries/libbom/Headers/bom/bom_format.h

```diff
@@ -30,6 +30,19 @@
 extern "C" {
 #endif
 
+#if _MSC_VER
```

Comment:
libbom is used standalone outside xcbuild so purposely has no dependencies. cc @marcsalem